config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/tool/tools_eu.properties

```diff
@@ -212,6 +212,8 @@ sakai.meetings.description= A tool for online meetings management
 sakai.meetings.title= Meetings
 microsoft.mediagallery.description= Ikusentzunezko fitxategiak bilatzeko Microsoft-en tresna bat da hau.
 microsoft.mediagallery.title= Galeria
+microsoft.collaborativedocuments.description= A Microsoft tool to browse Colaborative Documents
```

Comment:
@pablosanroman not in a hurry, you can provide these values whenever you want.

Thanks

Reply:
Hi @mpellicer  !
config.collaborative_documents_config=Lankidetzarako dokumentuen konfigurazioa

Reply:
microsoft.collaborativedocuments.description= Microsoften tresna bat lankidetzarako dokumentuak bilatzeko